compatibility-tests/v1_18/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![cfg(test)]
 
+extern crate core;
```

Comment:
I saw you said

> it can be supported if we put extern crate core; in the test, but that is a breaking change for everyone using 1.18

I wanted to see which versions would be affected, so I removed this line and built in 1.18, but I didn't get an error; any idea what might have changed or that I might be doing wrong?

Reply:
It's actually Rust 2015 that's affected. The proc macro targets core, but that's not in the prelude in Rust 2015.